examples/omp.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include <omp.h>
 
-#include <semiprof/semiprof.hpp>
+#include <semiprof.hpp>
```

Comment:
Keep the old version, and make the CMakeLists.txt so that the include path is `include`, not `include/semiprof`

Reply:
Thanks, should be changed now.